crossfit/dataset/base.py

```diff
@@ -16,7 +16,10 @@
 from dataclasses import dataclass
 from typing import Dict, Optional, Union
 
-import dask_cudf
+try:
+    from dask_cudf import read_parquet
+except ImportError:
+    from cudf import read_parquet
```

Comment:
If Dask is not available, then use cuDF to read a `class Dataset` object instead.